scripts/zones/Lower_Jeuno/npcs/Chululu.lua

```diff
@@ -1,115 +1,148 @@
-        player:startEvent(28); -- Start quest "Collect Tarut Cards" with option
-    elseif (CollectTarutCards == QUEST_ACCEPTED) then
-        player:startEvent(27); -- During quest "Collect Tarut Cards"
-    elseif (CollectTarutCards == QUEST_COMPLETED and RubbishDay == QUEST_AVAILABLE and player:getCharVar("RubbishDay_day") ~= VanadielDayOfTheYear()) then
---      prog = player:getCharVar("RubbishDay_prog");
+    if player:getFameLevel(JEUNO) >= 3 and CollectTarutCards == QUEST_AVAILABLE then
+        player:startEvent(28) -- Start quest "Collect Tarut Cards" with option
+    elseif CollectTarutCards == QUEST_ACCEPTED then
+        player:startEvent(27) -- During quest "Collect Tarut Cards"
+    elseif CollectTarutCards == QUEST_COMPLETED and RubbishDay == QUEST_AVAILABLE and player:getCharVar("RubbishDay_day") ~= VanadielDayOfTheYear() then
+--      prog = player:getCharVar("RubbishDay_prog")
 --      if (prog <= 2) then
---          player:startEvent(199); -- Required to get compatibility 3x on 3 diff game days before quest is kicked off
+--          player:startEvent(199) -- Required to get compatibility 3x on 3 diff game days before quest is kicked off
 --      elseif (prog == 3) then
-            player:startEvent(198); -- Start quest "Rubbish Day" with option
+            player:startEvent(198) -- Start quest "Rubbish Day" with option
 --      end
-    elseif (CollectTarutCards == QUEST_COMPLETED and RubbishDay == QUEST_AVAILABLE) then
-        player:startEvent(57); -- Standard dialog between 2 quests
-    elseif (RubbishDay == QUEST_ACCEPTED and player:getCharVar("RubbishDayVar") == 0) then
-        player:startEvent(49); -- During quest "Rubbish Day"
-    elseif (RubbishDay == QUEST_ACCEPTED and player:getCharVar("RubbishDayVar") == 1) then
-        player:startEvent(197); -- Finish quest "Rubbish Day"
-
-    elseif (SearchingForTheRightWords == QUEST_COMPLETED) then
-        if (player:getCharVar("SearchingForRightWords_postcs") < -1) then
-            player:startEvent(56);
+    elseif CollectTarutCards == QUEST_COMPLETED and RubbishDay == QUEST_AVAILABLE then
+        player:startEvent(57) -- Standard dialog between 2 quests
+    elseif RubbishDay == QUEST_ACCEPTED and player:getCharVar("RubbishDayVar") == 0 then
+        player:startEvent(49) -- During quest "Rubbish Day"
+    elseif RubbishDay == QUEST_ACCEPTED and player:getCharVar("RubbishDayVar") == 1 then
+        player:startEvent(197) -- Finish quest "Rubbish Day"
+    elseif player:getFameLevel(JEUNO) >= 4 and CollectTarutCards == QUEST_COMPLETED and AllInTheCards == QUEST_AVAILABLE then
+        player:startEvent(10110) -- Start quest "All in the Cards" with option
+    elseif AllInTheCards >= QUEST_ACCEPTED and player:getLocalVar("Cardstemp") == 0 then
+        if cdate >= os.time() then
+            player:startEvent(10111) -- During quest "All in the Cards" and same AllInTheCards_date value
```

Comment:
I think the only other issue I see is there is nothing in onEventFinish that sets `Cardstemp` for this CS, so after they accept the quest but before they complete it, while they're waiting, the other quests aren't accessible.

Reply:
Change made,  coco